service/src/main/resources/application.yml

```diff
@@ -77,6 +79,10 @@ profile:
     base-path: ${env.sam.basePath}
     admins-group-email: ${env.sam.adminsGroupEmail}
 
+  policy:
+    client-credential-file-path: build/resources/main/generated/bpm-client-sa.json
```

Comment:
Are there any helmfile changes that need to happen to get this file into place?

Reply:
I forgot the values in this config _are_ for local runs. Helmfile PR is here - https://github.com/broadinstitute/terra-helmfile/pull/4764